README.md

```diff
@@ -5,11 +5,20 @@ JSON diffing on steroids.
 ## 🧬 Installation
 
 ```shell
-npm install joey-the-differ
+npm install mawrkus/joey-the-differ
 ```
 
 ## 🧬 Usage
```

Comment:
I'd start with "CLI":

```shell
Usage: joey-the-differ [options]

Options:
  -V, --version        output the version number
  -s, --source [file]  source JSON file, required (default: "")
  -t, --target [file]  target JSON file, required (default: "")
  -h, --help           display help for command
```

But don't worry, I'll do this in the PR where the CLI options is added.

Reply:
Yep, I updated the README in #4